Sources/PackageModel/UserToolchain.swift

```diff
@@ -786,6 +873,40 @@ public final class UserToolchain: Toolchain {
         return .none
     }
 
+    private static func getWindowsPlatformInfo(
+        swiftSDK: SwiftSDK,
+        environment: Environment,
+        fileSystem: any FileSystem
+    ) -> (AbsolutePath, WindowsPlatformInfo)? {
+        let sdkRoot: AbsolutePath
+
+        if let sdkDir = swiftSDK.pathsConfiguration.sdkRootPath {
+            sdkRoot = sdkDir
+        } else if let SDKROOT = environment["SDKROOT"], let sdkDir = try? AbsolutePath(validating: SDKROOT) {
+            sdkRoot = sdkDir
+        } else {
+            return .none
+        }
+
+        // The layout of the SDK is as follows:
+        //
+        // Library/Developer/Platforms/[PLATFORM].platform/Developer/Library/<Project>-[VERSION]/...
```

Comment:
Do we really use `Library/Developer` directory naming on Windows? I thought this was very Darwin-specific

Reply:
I'd think so since this was the original comment I moved, that's how we discover XCTest today.